extension/lib/emailjs/emailjs-mime-codec.js

```diff
@@ -697,12 +697,22 @@
                 }
 
             } else {
-
                 // first line includes the charset and language info and needs to be encoded
                 // even if it does not contain any unicode characters
                 line = 'utf-8\'\'';
                 isEncoded = true;
                 startPos = 0;
+
+                // fix for attachments with emoji in filenames
+                if (typeof Intl !== 'undefined' && typeof Intl.Segmenter === 'function') {
+                    // Intl.Segmenter() currently not available on Firefox
+                    // https://caniuse.com/mdn-javascript_builtins_intl_segmenter
+                    encodedStr = [...new Intl.Segmenter().segment(encodedStr)].map(x => x.segment)
+                } else {
+                    // regex from https://stackoverflow.com/a/69661174/3091318
+                    encodedStr = encodedStr.replace(/(?![*#0-9]+)[\p{Emoji}\p{Emoji_Modifier}\p{Emoji_Component}\p{Emoji_Modifier_Base}\p{Emoji_Presentation}]/gu, '')
+                }
```

Comment:
This is nice - I tried to find a reference for regex in Kotlin but I do not see any relevant documentation/reference material that Kotlin supports \p{Emoji_Presentation} or similar, unlike javascript which has rich support with it. So with that, we might end up using emoji-unicode ranges which again may contain an incomplete range of emojis. 

@sosnovsky Do you probably allow if I fix both #5308 and #5309 by re-using the regex on line 713?

Reply:
Yeah, I tried to use the same regex on FES for allowing emojis, but unfortunately Kotlin doesn't support these `\p` keywords.

Let me finish this PR, then I'll check possible solution for #5308 and #5309 on FES side, and if there won't be a simple way to allow emojis and other symbols in recipient names - I'll notify you that you can work on stripping emojis from recipient names on browser extension side.